docs/modules/publisher.md

```diff
@@ -10,7 +10,7 @@ It runs on [Azure IoT Edge](https://azure.microsoft.com/en-us/services/iot-edge/
 
 Please use our released containers for OPC Publisher available in the Microsoft Container Registry, rather than building from sources. The easiest way to deploy OPC Publisher is through the [Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/microsoft_iot.iotedge-opc-publisher). 
 
-[<img src="image-20201028141833399.png" style="zoom:50%;" />](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/microsoft_iot.iotedge-opc-publisher)
```

Comment:
The image does not seem to be present. Please check this document:

https://github.com/Azure/Industrial-IoT/blob/mschier%2Fopsmanual/docs/modules/publisher.md

![image](https://user-images.githubusercontent.com/20900945/122887434-2f9bb280-d341-11eb-8069-4412f42bf1b1.png)

Reply:
Correct, let me see if I can dig it up from the publisher repo.

Reply:
Done.